src/uct/ib/ud/base/ud_iface.c

```diff
@@ -659,6 +673,10 @@ ucs_config_field_t uct_ud_iface_config_table[] = {
      "Max number of receive completions to pick during asynchronous TX poll",
      ucs_offsetof(uct_ud_iface_config_t, rx_async_max_poll), UCS_CONFIG_TYPE_UINT},
 
+    {"TX_CQ_MODERATION", "64",
```

Comment:
why need to make it configurable?

Reply:
With EFA, `IBV_SEND_SIGNALED` must be set in send flags. Otherwise, `ibv_post_send()` will fail. So, CQ moderation must be set to 1 for EFA. But, maybe I should detect the device type and set it to 1 if it is EFA.

Reply:
yes, and let's make sure it's only in verbs code (not in common) so will not affect mlx5